Makefile

```diff
@@ -0,0 +1,19 @@
+up:
+	docker-compose -f dev.yml up -d;
+	./setup_pubsub.sh
+	
+down:
+	docker-compose -f dev.yml down
+
+logs:
+	docker-compose -f dev.yml logs --follow
+
+unit-test:
+	go test -race ./processor/./...
+
+int-test:
+	docker-compose -f dev.yml up -d;
+	./setup_pubsub.sh
+	PUBSUB_EMULATOR_HOST=localhost:8539 go test *.go
```

Comment:
I've had this fail intermittently in my shell with error 
```
2020/04/20 10:40:36 Failed to connect to RabbitMQ: Exception (501) Reason: "EOF"
FAIL	command-line-arguments	0.185s
FAIL
make: *** [int-test] Error 1
```
I suspect it's trying to run the test before rabbit has quite started up?

Reply:
yes that's prob whats happening, i seem to remember jamie put something to wait for things to come up in the python version

Reply:
try this: `bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' '$PUBSUB_SETUP_HOST')" != "200" ]]; do sleep 1; done'`

Reply:
Yeah, fine to leave it for now but I wanted to note it because it'll need sorting if we want reliable travis builds as and when we implement them.

Reply:
@AdamHawtin i've cribbed what we have in the python version so this should be fixed now.....